CSS/index.css

```diff
@@ -106,4 +111,103 @@ input#menuButton {
     transform: translateY(-305px);
     transition: transform 500ms;
   }
+
+  #bGImage {
+    margin-top: -213px;
+  }
+}
+/* Main Banner */
+#bGImage {
+  background-image: url("../assets/BGimage.png");
+  background-color: rgb(241, 241, 241);
+  height: 700px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  z-index: -1;
+
+  /* is this negative margin ok? */
+}
+
+.b_g_text {
+  text-align: center;
+  position: absolute;
+  font-size: 40px;
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
Not using the google font?!

Reply:
I still don't understand how to do that.